### DISCUSSION TITLE
Questions on script  to update synology compatible db - looking for some clarification

### DISCUSSION THREAD

**Discussion by sbrews** [007revad/Synology_HDD_db#193] [Q&A] [answered]
I recently acquired a DS1823xs+ with some WD Red Plus disk and 2 Crucial PCIe SSD disk... and got the irritating "not verified" status.  Looking at this script to update the compatibility DB, I have some questions:

Do I just put the script on the NAS and run it?

Or do I need to figure out the vendor ID info for the above disks/ssd, add that to the vendor file, put it with script and THEN run the script?  

And for what may be the dumb question of the day, how/where do I determine the vendor info for the above if it needs to be in the text file before running the script?

I did read the read-me for the script, but the above didnt seem to be addressed... if this is documented someplace and I missed it, please point me to where to find this particular detail.

**Comment by 007revad** [comment, marked as answer]
[Download the zip or tar.gz file](https://github.com/007revad/Synology_HDD_db/releases/tag/v3.3.72) to a shared folder on your NAS, then extract the zip or tar.gz file (you can use File Station) and run the script.

I only added the syno_hdd_vendor_ids.txt file 2 weeks ago. It looks like I need to improve the documentation to better explain the purpose of the syno_hdd_vendor_ids.txt file.
1. The syno_hdd_vendor_ids.txt file is only needed if DSM doesn't know the brand of your NVMe drives.
2. In this case the script will show you the vendor ID and advise you to add it to the syno_hdd_vendor_ids.txt file.

**Reply by 007revad** [reply]
I've updated the readme with information on what the vendor ids txt file is for and when someone would need to edit it. 
https://github.com/007revad/Synology_HDD_db#vendor_idstxt
